crates/script/src/runner.rs

```diff
@@ -165,6 +172,7 @@ impl ScriptRunner {
 
         // Optionally call the `setUp` function
         let (success, gas_used, labeled_addresses, transactions) = if !setup {
+            // self.executor.backend_mut().set_test_contract(address);
```

Comment:
I introduce  this because it's the upstream code and can give context when troubleshooting or solving merge conflicts, it's paired with the comments above

Reply:
Please also add a small comment here for context